tidb-server/main.go

```diff
@@ -112,5 +112,9 @@ func main() {
 		os.Exit(0)
 	}()
 
+	go systimeMonitor(time.Now, func() {
+		log.Fatal("error: system time jump forward")
```

Comment:
So after the first time we meet time jump forward, the monitor go routine will exit?